lib/friends/introvert.rb

```diff
@@ -212,6 +212,27 @@ def add_nickname(name:, nickname:)
       @output << "Nickname added: \"#{friend}\""
     end
 
+    # Add an alias to an existing location.
+    # @param name [String] the name of the location
+    # @param nickname [String] the alias to add to the location
+    # @raise [FriendsError] if 0 or 2+ location match the given name
+    # @raise [FriendsError] if the alias is already taken
+    def add_location_alias(name:, nickname:)
```

Comment:
Would you mind changing these method names to `add_alias` and `remove_alias` to more closely match the CLI commands?

Reply:
Right, i forgot to change their names after the discussion about how the command should be named.